kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KopEventManager.java

```diff
@@ -242,18 +251,26 @@ private JsonObject parseJsonObject(String info) {
     }
 
     @VisibleForTesting
```

Comment:
Maybe it's a miss in previous review. Why did you add `@VisibleForTesting` here? I see `getNodes` is only called in private method `KopEventManager#getBrokers`.

Reply:
Or should we add tests for this method?

Reply:
Sorry it's a problem with my Idea (maybe cache was not cleared), resolve it.